README.md

```diff
@@ -1,6 +1,20 @@
 [![ROS Industrial
 CI](https://github.com/fzi-forschungszentrum-informatik/cartesian_ros_control/actions/workflows/industrial_ci_action.yml/badge.svg?branch=master&event=push)](https://github.com/fzi-forschungszentrum-informatik/cartesian_ros_control/actions)
 
+---
+##### ++ Deprecation Notice ++
+
+This repository has been split and restructured into individual packages to
+facilitate upstream merges into ROS mainline repositories.
```

Comment:
This would get it a bit more visual support:
```suggestion
<mark>This repository has been split and restructured into individual packages to
facilitate upstream merges into ROS mainline repositories.</mark>
```

Reply:
@fmauch Should I see a difference for this additional tag? It looks identical (for me) on the rendered github page...

Reply:
Maybe github doesn't support this, then... Then, https://github.com/fzi-forschungszentrum-informatik/cartesian_ros_control/pull/30#discussion_r649880537 has to be enough.